src/Resources/contao/library/Contao/Database/Statement.php

```diff
@@ -375,43 +355,11 @@ protected function escapeParams($arrValues)
 	/**
 	 * Debug a query
 	 *
-	 * @param \Database\Result $objResult An optional result object
+	 * @deprecated Deprecated in Contao 4.0, to be removed in Contao 5.0.
```

Comment:
On the other hand, IMO you can also remove it, as the compatibility has changed anyway as you removed the abstract protected methods below and the descendant classes implementing them removed.
It was protected and only called internally, I do not see any usecase where any developer would end up calling it from his code.